src/elements/DoubleSlider/DoubleSlider.vue

```diff
@@ -0,0 +1,183 @@
+
+  library.add(faKeyboard)
+
+  /**
+   * ### Inputs fields accept user input, duh!
+   *
+   * Input fields form the basis of forms, the primary method of receiving input
+   * from the user, be it text, email addresses, passwords, dates or numbers.
+   */
+  export default {
+    name: 'InputField',
+    components: {
+      FontAwesomeIcon
+    },
+    mixins: [
+      Colored,
+      Indicating,
+      Joined,
+      Rounded,
+      Scaled,
+      Simplified,
+
+      Invertible,
+      Unactionable
+    ],
+    inheritAttrs: false,
+    model: {
+      prop: 'value',
+      event: 'input'
+    },
+    props: {
+      /**
+       * _an array specifying the left and right icon to use as add-ons_
+       *
+       * Use '' to omit the icon.
+       */
+      iconSet: {
+        type: Array,
+        validator: val => val.length === 2,
+        default: () => [null, null]
+      },
+      /**
+       * _whether to pad the left and right add-ons_
+       */
+      isAddonPaddedSet: {
+        type: Array,
+        validator: val => val.length === 2,
+        default: () => [true, true]
+      },
+      /**
+       * _whether to increase the coloration of the input field_
+       */
+      isInfused: {
+        type: Boolean,
+        default: false
+      },
+      /**
+       * _this is the renamed `size` attribute from the HTML spec_
+       */
+      charCount: {
+        type: Number,
+        default: 1
+      }
+    },
+    computed: {
+      inputFieldClasses: function () {
+        return [
+          ...this.coloredClasses,
+          ...this.indicatingClasses,
+          ...this.joinClasses,
+          ...this.roundedClasses,
+          ...this.scaledClasses,
+          ...this.simplifiedClasses,
+
+          ...this.invertibleClasses,
+          ...this.unactionableClasses,
+
+          {
+            'infused': this.isInfused
+          }
+        ]
+      },
+      fieldClasses: function () {
+        return [
+          {
+            'has-left-addons': this.hasLeftAddons,
+            'has-right-addons': this.hasRightAddons
+          }
+        ]
+      },
+      leftAddonClasses: function () {
+        return [
+          {
+            'padded': this.isAddonPaddedSet[0]
+          }
+        ]
+      },
+      rightAddonClasses: function () {
+        return [
+          {
+            'padded': this.isAddonPaddedSet[1]
+          }
+        ]
+      },
+
+      hasLeftAddons: function () {
+        return this.iconSet[0] || this.$slots.leftAddons
+      },
+      hasRightAddons: function () {
+        return this.iconSet[1] || this.$slots.rightAddons
+      }
+    },
+    methods: {
+      emitInput: function (event) {
+        /**
+         * _event emitted when input given to field_
+         *
+         * The value of the field is passed back.
+         *
+         * @type {string}
+         */
+        this.$emit('input', event.target.value)
+      }
+    }
+  }
+</script>
+
+<style lang="stylus" src="./InputField.styl">
```

Comment:
The name of the stylesheet is incorrect which is also responsible for the Netlify preview build failing.